Gopkg.lock

```diff
@@ -259,6 +363,12 @@
   packages = ["."]
   revision = "a61a99592b77c9ba629d254a693acffaeb4b7e28"
 
+[[projects]]
+  branch = "master"
+  name = "github.com/mitchellh/hashstructure"
+  packages = ["."]
+  revision = "2bca23e0e452137f789efbc8610126fd8b94f73b"
+
```

Comment:
I have a separate PR for adding this - it was my mistake it got left out, but I'll just merge yours I think.